src/models/encryption-model.ts

```diff
@@ -0,0 +1,20 @@
+import { Model } from "sequelize";
+import { CryptorHttpClient, CryptorSecretKey  } from "../util/cryptor-http-client";
+import Logger from 'bunyan';
+
+export abstract class EncryptionModel<TSecretFields> extends Model{
```

Comment:
I'd rather call this `EncryptedModel`, because it's not a model of an encryption, but rather a model of which parts are encrypted.